sim/team-validator.ts

```diff
@@ -2519,9 +2519,18 @@ export class TeamValidator {
 						} else if (level >= 5 && learnedGen === 3 && species.canHatch) {
 							// Pomeg Glitch
 							learned = learnedGen + 'Epomeg';
-						} else if ((!species.gender || species.gender === 'F') &&
+						} else if (species.gender !== 'N' &&
 							learnedGen >= 2 && species.canHatch && !setSources.isFromPokemonGo) {
 							// available as egg move
+							if (species.gender && species.gender === 'M') {
```

Comment:
```suggestion
							if (species.gender === 'M') {
```
Similarly, `'M'` can't be falsy either.